Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -329,16 +472,15 @@ namespace AzToolsFramework
                     }
                     else if (operationIterator->GetType() == AZ::Dom::PatchOperation::Type::Remove)
                     {
-                        rowLayout->removeWidget(rowWidget);
+                        m_layout->removeWidget(rowWidget);
                         delete rowWidget;
                     }
                 }
             }
             else
             {
                 // delegate the action th the rowWidget, which will, in turn, delegate to the next row in the path, if available
-                auto rowWidget =
-                    static_cast<DPERowWidget*>(GetVerticalLayout()->itemAt(static_cast<int>(firstAddressEntry.GetIndex()))->widget());
+                auto rowWidget = static_cast<DPERowWidget*>(m_layout->itemAt(static_cast<int>(firstAddressEntry.GetIndex()))->widget());
```

Comment:
nit: prefer `aznumeric_cast`

Reply:
thanks, and done! I replaced a ton of those now.